.circleci/build_image.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+set -e
+
+if [[ $# -ne 2 ]]; then
+  echo "Usage: $0 DOCKERFILE IMAGE_NAME"
+  echo ".dockerfile is appended to DOCKERFILE."
+  echo "janitortechnology/ is prepended to IMAGE_NAME."
```

Comment:
Nit: Please don't append or prepend anything to parameters. It makes things opaque and confusing (e.g. "What is chromium/chromium?") and it reduces flexibility (e.g. what if we want to build an importer Dockerfile that doesn't have a `.dockerfile` extension? Or what if we want to tag some image with a different prefix? Etc.)